nodl_python/nodl/_parsing/_parsing.py

```diff
@@ -65,3 +64,26 @@ def parse(path: Union[str, Path, IO]) -> List[Node]:
     element_tree = etree.parse(path)
 
     return _parse_element_tree(element_tree)
+
+
+def parse_multiple(paths: Sequence[Union[str, Path, IO]]) -> List[Node]:
```

Comment:
Can you explain the use-case behind such a function?

Reply:
Moved to new pr